src/script/components/calling/GroupVideoGridTile.tsx

```diff
@@ -105,8 +108,29 @@ const GroupVideoGridTile: React.FC<GroupVideoGridTileProps> = ({
     }
   };
 
+  const handleZoomClick = () => {
+    if (isZoomedIn) {
+      setIsZoomedIn(false);
+    } else {
+      setIsZoomedIn(true);
+    }
+  };
+
   const participantNameColor = getParticipantNameColor({isActivelySpeaking, isAudioEstablished});
 
+  const actionItem = !minimized && sharesScreen && (
+    <button
+      data-uie-name="item-grid"
+      data-user-id={participant?.user.id}
+      className="group-video-grid__element__action_icon"
+      onClick={handleZoomClick}
+      onKeyDown={handleZoomClick}
```

Comment:
two are unnecessarily, only onClick should be enough

Reply:
You use both events in many places... I thought it was a workaround.  I will remove it